Modules/Core/Common/include/itkImageRegion.h

```diff
@@ -378,6 +413,35 @@ std::ostream &
 operator<<(std::ostream & os, const ImageRegion<VImageDimension> & region);
 } // end namespace itk
 
+
+namespace std
+{
+// NOLINTBEGIN(cert-dcl58-cpp)
+// Locally suppressed the following warning from Clang-Tidy (LLVM 17.0.1), as it appears undeserved.
+// > warning: modification of 'std' namespace can result in undefined behavior [cert-dcl58-cpp]
```

Comment:
@dave3d Hi Dave! Can you please tell me why Spell Checking Code Comments is complaining? It says at https://github.com/InsightSoftwareConsortium/ITK/actions/runs/7186016833/job/19570520532?pr=4367

>    Modules/Core/Common/include/itkImageRegion.h, 419
>
>   Modules/Core/Common/include/itkImageRegion.h, 421
>   Modules/Core/Common/include/itkImageRegion.h, 442
> 3 misspellings found

How can we see exactly which specific misspellings it refers to?

With this commit, I did already add `NOLINTBEGIN`, `NOLINTEND`, and `cert-dcl58-cpp` to ".github\workflows\itk_dict.txt". Shouldn't that already make the spell checker happy?

Reply:
I'm not sure.  I ran it on my computer and it passed OK.

Reply:
It seems like it could be using the old, non-updated dictionary?

Reply:
Thanks for your feedback @dave3d Indeed, the latest revision passes OK, after some trial and error! Apparently because I added `cert`, `dcl`, and `cpp` as three separate terms to the dictionary, instead of `cert-dcl58-cpp` as a whole! (This [force-push](https://github.com/InsightSoftwareConsortium/ITK/compare/2e9874271c6cfb4a4559d9bde61ac1cc6d7b053a..5cf540ef0759dbf37c97b8b4761b30423cc5815a) and this [force-push](https://github.com/InsightSoftwareConsortium/ITK/compare/5cf540ef0759dbf37c97b8b4761b30423cc5815a..c8c9983a697c97d5f032d57cd1656c15b6a64be4).)

Problem resolved 👍

Reply:
@dave3d Can you please have a look at the related CommentSpellCheck issues that I just submitted?
- https://github.com/SimpleITK/CommentSpellCheck/issues/53
- https://github.com/SimpleITK/CommentSpellCheck/issues/54

Reply:
OK, I can fix the code to do those things.